wgengine/pendopen.go

```diff
@@ -106,6 +108,17 @@ func (e *userspaceEngine) trackOpenPostFilterOut(pp *packet.Parsed, t *tstun.TUN
 	}
 
 	flow := flowtrack.Tuple{Src: pp.Src, Dst: pp.Dst}
+
+	// iOS likes to probe Apple IPs on all interfaces to check for connectivity.
+	// Don't start timers tracking those. They won't succeed anyway. Avoids log spam
+	// like:
+	//    open-conn-track: timeout opening (100.115.73.60:52501 => 17.125.252.5:443); no associated peer node
+	if runtime.GOOS == "ios" && flow.Dst.Port == 443 && !tsaddr.IsTailscaleIP(flow.Dst.IP) {
+		if _, ok := e.magicConn.PeerForIP(flow.Dst.IP); !ok {
```

Comment:
Won't this catch on someone attempting to access a web server on a shared subnet?

Reply:
no, because the `!ok`

Reply:
we only skip the timer if it's port 443 on iOS going to a non-Tailscale IP for which we have no route from any peer

Reply:
oh I misread that as `ok`